### ISSUE TITLE
[SSC] Tidewalker Hydromancer - doubled damage?

### ISSUE THREAD

**Issue by icywire** [SunstriderEmu/BugTracker#4283]
**Describe the bug**
Tidewalker Hydromancer does doubled damage from spell abilities. 
Frost Shock and Frostbolt hit for 5-6k.

**To Reproduce**

1. Fight with Tidewalker Hydromancer

Frost Shock: https://db.endless.gg/?spell=39062
Frostbold: https://db.endless.gg/?spell=39064

**Expected behavior**
Spells should hit for damage as described in links above.

**Screenshots/videos**
Don't have

**Additional context**
https://wowwiki.fandom.com/wiki/Tidewalker_Hydromancer?oldid=709103 - first post on wowwiki with dmg
http://sisuguild.fi/~archsin/wws/20070903-koko_raidi/tidewalkerhydromancer.html - real stats from retail raid

**Comment by Jerrythemagicdog** [comment]
Yes, damage of every ability in every raid on every creature and boss is changed.

**Comment by icywire** [comment]
I'm perfectly fine with damage changed in compare to retail - this rule applies to every private server.
But please help me understand something different. I wonder if site db.endless.gg shows 'real' values that may occur while playing on endless server? Is there some kind of multiplier that is taken into account in game, but not in db?

In this particular case damage that we saw in game was 2x higher than values in db.endless.gg
It may mean that either db has some bug or server has some bug.

**Comment by Jerrythemagicdog** [comment]
db.endless.gg just shows default client values, they're not consistent with in-game values.
